cli/src/commands/print.rs

```diff
@@ -1,33 +0,0 @@
-use crate::commands::Command;
```

Comment:
can we not lose this command? we can rename it or put it under a `debug` or `experimental` or something, but i've been using it a lot to look at how the parser is parsing things